projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -29,9 +29,9 @@ deploymentBuilderImage:
 
 deploymentBuilder:
   securityContext:
-    runAsUser: 0
-    runAsGroup: 1000
-    fsGroup: 1000
+    runAsUser: "0"
```

Comment:
Where is the "if 0" check? If possible, is it not better to convert values to string there, before the check, so that cases like this are avoided in the future?

Reply:
https://github.com/vmware/versatile-data-kit/blob/main/projects/control-service/projects/helm_charts/pipelines-control-service/templates/deployment.yaml#L219

Reply:
It's debatable